templates/profile.html

```diff
@@ -21,6 +21,25 @@ <h3>
                         second name: {{profile.last_name}} <br>
                         age: {{profile.age}} <br>
                         talents: {{profile.balance}}T <br>
+                        {% if opinions %}
+                            {% regroup opinions by opinion as opinion_list %}
+                            {% for opinion in opinion_list %}
+                                <div class="card mb-3 bg-dark">
+                                    <div class="card-body">
+                                        {{ opinion.grouper }}
```

Comment:
Nie mam pojęcia dlaczego to nie działa (wyświetla None zamiast każdej kategorii w rezultacie nie ma żadnego grupowania)

Reply:
Zamień na:
{% regroup opinions by category as opinion_list %}